event-adapter/pkg/apiserver/installer/installer.go

```diff
@@ -186,85 +184,36 @@ func (a *MetricsAPIInstaller) registerResourceHandlers(storage rest.Storage, ws
 		}
 
 		ctx = request.WithUserAgent(ctx, req.HeaderParameter("User-Agent"))
-
-		// inject the resource, subresource, and name here so that
```

Comment:
You don't have to remove this comment, it still holds true. Please talk to me if you want me to explain it further.